samples/RiteKit.gs

```diff
@@ -0,0 +1,53 @@
+/*
+ * This sample demonstrates how to configure the library for the RiteKit API.
+ * Instructions on how to generate OAuth credentuals is available here:
+ * https://ritekit.docs.apiary.io/#introduction/options-for-authorizing-api-calls
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+/**
+ * Authorizes and makes a request to the RitKit API.
+ */
+function run() {
+  var service = getService();
+  if (service.hasAccess()) {
+    console.log(service.getAccessToken());
```

Comment:
Please remove this debugging line.

Reply:
Can't understand why is `no-console` of lint not raised.